index.html

```diff
@@ -0,0 +1,50 @@
+<!-- @format -->
+
+<!DOCTYPE html>
+<html lang="en" class="html">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@200&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+  </head>
+
+  <body>
+    <header class="upper">
+      <div class="toolbar">
+        <p id="name">Yilikal Teshome</p>
+        <nav class="drop">
+          <img id="icondrop" src="dropbox.png" alt="" />
+        </nav>
+      </div>
+    </header>
+    <section class="lower">
+      <div>
+        <h1>Hey There. I'm Yilikal</h1>
+        <h2>I'm a Software Developer</h2>
+        <p class="detail">
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
+        </p>
+        <div class="sidepic">
+          <img src="Illustration(1).png" alt="" />
+        </div>
+      </div>
+      <div>
+        <nav class="icons">
+          <img src="github.png" alt="" />
+          <img src="linked-in-logo-of-two-letters.png" alt="" />
+          <img src="peace.png" alt="" />
+          <img src="twitter(1).png" alt="" />
+          <img src="m.png" alt="" />
+        </nav>

```

Comment:
- [ ] Kindly use the ul and li, a tags for the social media icons, it should look like this:
 `<li><a href="#"><img src="m.png" alt="medium" /></a></li>`

- [ ] Please add alt text to all images this is important so when browsers are unable to render a page properly a text is displayed 👍🏿 


 Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![dd](https://user-images.githubusercontent.com/69638013/159033598-f79ee316-009f-4947-b51d-4c68781ca3f8.png)  | ![ggg](https://user-images.githubusercontent.com/69638013/159033514-f511c5d7-f2b7-4c84-89cf-fdcaba84de8b.png)

- [ ] Kindly remove background color from the social media icons